features/step_definitions/journey_test_steps.rb

```diff
@@ -707,12 +731,18 @@
   current_url.should end_with("#{dm_frontend_domain}/admin/#{page_name.downcase}?supplier_name_prefix=#{supplier_search_prefix.gsub(' ','+')}")
   page.should have_selector(:xpath, "//table/tbody/tr[1]/td[1]//*[contains(text(),'#{supplier_search_prefix}')]")
   page.find(:xpath,
+    "//table/tbody/tr[1]//*[contains(text(),'DM Functional Test Supplier')]/../..//a[contains(@href,'/admin/suppliers/11111/edit/name')][text()]"
+  ).text().should have_content('Change name')
```

Comment:
I find these really hard to understand. Maybe not for this pull request, but it would be good to think about how they could be abstracted behind some functions.